resources/user-agents/browsers/mobile-safari/mobile-safari-10-0-on.json

```diff
@@ -30,62 +30,33 @@
         {
           "match": "Mozilla/5.0 (#DEVICE##PLATFORM#)*applewebkit*(*khtml*like*gecko*)*Version/#MAJORVER#.#MINORVER#*Safari/*",
           "devices": {
-            "iPad": "iPad",
             "iPhone": "iPhone",
             "iPod": "iPod Touch"
           },
           "platforms": [
-            "iOS_A_8_0", "iOS_A_8_0_B",
-            "iOS_A_8_1", "iOS_A_8_2", "iOS_A_8_3", "iOS_A_8_4", "iOS_A_9_0", "iOS_A_9_1", "iOS_A_9_2", "iOS_A_9_3",
-            "iOS_A_10_0", "iOS_A_10_1", "iOS_A_10_2", "iOS_A_10_3",
+            "iOS_A_10_3", "iOS_A_10_2", "iOS_A_10_1", "iOS_A_10_0",
+            "iOS_A_9_3", "iOS_A_9_2", "iOS_A_9_1", "iOS_A_9_0",
+            "iOS_A_8_4", "iOS_A_8_3", "iOS_A_8_2", "iOS_A_8_1", "iOS_A_8_0",
             "iOS_A",
-            "iOS_B",
-            "iOS_C_8_4", "iOS_C_8_3", "iOS_C_8_2", "iOS_C_8_1", "iOS_C_8_0", "iOS_C_9_3", "iOS_C_9_2", "iOS_C_9_1", "iOS_C_9_0",
-            "iOS_C_10_3", "iOS_C_10_2", "iOS_C_10_1", "iOS_C_10_0",
             "iOS_C"
```

Comment:
I left the version-less iOS_C platform in the iphone/ipod sections for Mobile Safari because it was one of the browsers that I did see some cases of the iOS_C pattern with iPhone.  I didn't keep all of the version numbers though, as I think these patterns are still suspect (and relatively rare) so I don't think it's worth the extra patterns/maintenance.